code/ship/ship.cpp

```diff
@@ -15808,7 +16466,7 @@ void ship_page_in()
 				for (j = 0; j < sip->n_subsystems; j++)
 					sip->subsystems[j].model_num = -1;
 
-				ship_copy_subsystem_fixup(sip);
+				ship_copy_subsystem_fixup(&(*sip));
```

Comment:
This is the correct idiom.  It appears elsewhere in our codebase and I've seen it used in other places online as well.